### ISSUE TITLE
FR: Feed cfn-python-lint with JSON Schema of Third-Party Cloudformation resources

### ISSUE THREAD

**Issue by azec-pdx** [aws-cloudformation/cfn-python-lint#1401]
**TLDR;**  *Add support to import Third-Party CF Resource  JSON Schema specification into `cfn-lint` (and not just AWS Resources)*
<hr>

**cfn-lint version: (`cfn-lint --version`)**

```
cfn-lint --version
cfn-lint 0.28.3
```

**Description of issue.**

`Feature request:`
* I am trying to use recently announced [Third-Party resources in CloudFormation](https://aws.amazon.com/blogs/aws/cloudformation-update-cli-third-party-resource-support-registry/).
* The early adopter on this concept was DataDog, and resource I am trying to use is [Datadog::Monitors::Monitor](https://github.com/DataDog/datadog-cloudformation-resources/tree/master/datadog-monitors-monitor-handler) in one of my CloudFormation stacks.
* After importing this TP resource JSON Schema specification in CloudFormation Registry, had no issues stacking stacks with `Datadog::Monitors::Monitor` resource within. 
* My IDE (Pycharm) and my CI/CD both use `cfn-lint`. My CI/CD pipeline uses `cfn-lint` for validation in the build phase. 
* I would like to extend `cfn-lint` to:
    * not throw errors for unknown (Third-Party) resource, such as below:
       <img width="1124" alt="Screen Shot 2020-03-05 at 1 10 29 PM" src="https://user-images.githubusercontent.com/1272830/76025964-c80aeb00-5ee2-11ea-94d4-f6201126a57d.png">
    * not fail build in CI/CD pipeline because it doesn't recognize `Datadog::Monitors::Monitor` (or any other Third-Party) resource.
* The schema of the resource I am interested in (`Datadog::Monitors::Monitor`) is published at this [repo link](https://github.com/DataDog/datadog-cloudformation-resources/blob/master/datadog-monitors-monitor-handler/datadog-monitors-monitor.json).

I am curious if this is currently supported and how can I pull that schema in ? Is this configurable via 
`cfn-lint --update-specs` or will I need to re-build `cfn-lint` by feeding it this TP JSON Schema? Is that even supported? 

>Cfn-lint uses the [CloudFormation Resource Specifications]
>(https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) >as the base to do validation. These files are included as part of the application version. Please update to >the latest version of `cfn-lint` or update the spec files manually (`cfn-lint -u`)

Duh, how do I expand them or add external JSON Schema for non-AWS resources...?

Thanks!

**Comment by azec-pdx** [comment]
Not sure if appropriate label is `spec bug` ?

**Comment by PatMyron** [comment]
being tracked in https://github.com/aws-cloudformation/cfn-python-lint/issues/1277
